docs/tutorials/excel-tutorial.md

```diff
@@ -110,13 +110,15 @@ Let's convert this fruit sales data into a table. We'll use our script for the e
       selectedSheet.getRange("A2:C2").format.fill.color = "FFC000";
       selectedSheet.getRange("A3:C3").format.fill.color = "yellow";
       let table = selectedSheet.tables.add("A1:C5", true);
-      table.style = "TableStyleLight21";
+      table.sort.apply([{ key: 0, ascending: true }]);
     }
     ```
 
+    Tables have a `TableSort` object, accessed through the `Table.sort` property. You can apply sorting criteria to that object. The `apply` method takes in an array of `SortField` objects. In this case, we only have one sorting criteria, so we only use one `SortField`. `key: 0` sets the column with the sort-defining values to "0" (which is the first column on the table, **A** in this case). `ascending: true` sorts the data in ascending order (instead of descending order).
```

Comment:
Tangent - should table APIs also support keying by column name?

Reply:
I think that would be an awesome feature. It would be easier and help enable more relative-positioning scenarios.